### DISCUSSION TITLE
Welcoming Daniel Anner to the maintainers team!

### DISCUSSION THREAD

**Discussion by jeremystretch** [netbox-community/netbox#11636] [Announcements]
Please join me in welcoming @danner26 to the NetBox maintainers team! Dan has been absolutely **on fire** lately pushing through a ton of backlogged PRs in the [device types library](https://github.com/netbox-community/devicetype-library), and has recently agreed to join the core maintainers team. A bit more about him:

> Dan is currently an Infrastructure Engineer with both professional and passion project software development experience. He has a diverse range of experience from working as a SysAdmin in the Higher Education sector, professional software development for the FAA and running his own Managed IT Service Provider.

Welcome @danner26, we're thrilled to have you on the team!

**Comment by danner26** [comment]
Thanks for the warm welcome! I am excited to be here

**Comment by arthanson** [comment]
Welcome Daniel!

**Comment by jsenecal** [comment]
Yayy :partying_face: ! Welcome aboard @danner26 !!

**Comment by bitcollector1** [comment]
Welcome Daniel.
